src/test/scala/org/broadinstitute/dsde/firecloud/service/PassthroughDirectivesSpec.scala

```diff
@@ -179,24 +129,38 @@ class PassthroughDirectivesSpec extends BaseServiceSpec with HttpServiceBase
     }
   }
 
-  private def validateAllPaths(path:String, queryParams:Option[Map[String,String]]=None)(implicit ourTheirRoots:(String,String)): Unit = {
+  private def validateAllPaths(path: String, queryParams: Option[Map[String, String]] = None)
+                              (implicit ourTheirRoots: (String, String)): Unit = {
     validateAllPaths(path, path, queryParams)
   }
 
-  private def validateAllPaths(inpath:String, outpath:String)(implicit ourTheirRoots:(String,String)): Unit = {
+  private def validateAllPaths(inpath: String, outpath: String)
+                              (implicit ourTheirRoots: (String, String)): Unit = {
     validateAllPaths(inpath, outpath, None)
   }
 
-  private def validateAllPaths(inpath:String, outpath:String, queryParams:Option[Map[String,String]])
-                              (implicit ourTheirRoots:(String,String)): Unit = {
-    validateAllPathsWithRoots(inpath,outpath,queryParams,ourTheirRoots._1,ourTheirRoots._2)
+  private def validateAllPaths(inpath: String, outpath: String, queryParams: Option[Map[String, String]])
+                              (implicit ourTheirRoots:(String, String)): Unit = {
+    validateAllPathsWithRoots(inpath, outpath, queryParams, ourTheirRoots._1, ourTheirRoots._2)
   }
 
-  private def validateAllPathsWithRoots(inpath:String, outpath:String, queryParams:Option[Map[String,String]],
-                                       inroot:String, outroot:String): Unit = {
-    val actualInRoot =    if (inroot.equals("")) "" else "/"+inroot
-    val expectedOutRoot = if (outroot.equals("")) "" else "/"+outroot
-    val specRoute = passthroughAllPaths(inroot, echoUrl + expectedOutRoot)
+  private def validateAllPathsWithRoots(inpath: String, outpath: String,
+                                        queryParams: Option[Map[String, String]],
+                                        inroot: String, outroot: String): Unit = {
+    val actualInRoot = if (inroot.equals("")) "" else "/" + inroot
+    val expectedOutRoot = if (outroot.equals("")) "" else "/" + outroot
+
```

Comment:
@davidangb I pretty much included the code that used to be in `passthroughAllPaths()` below as part of `val specRoute = ...`. If I missed something or there's a better way to do it, let me know.

Reply:
where are the `validateAllPaths` functions actually used now? Since you're removing the `"passthroughAllPaths() directive"` test, I don't see them in use and these can just be deleted (therefore answering your question)

Reply:
Great, thanks!